pages/shared.jsx

```diff
@@ -0,0 +1,28 @@
+import React, {useEffect} from 'react';
+import SharedTop from "../../components/Shared/Top";
+import SharedContents from "../../components/Shared/SharedContents";
+import {getFolderSampleList} from "../../utils/api";
+import * as S from "../styles/Shared.style";
+
+export default function Shared() {
+  const [data, setData] = React.useState(null);
```

Comment:
```suggestion
import {useState, useEffect} from 'react';
import SharedTop from "../../components/Shared/Top";
import SharedContents from "../../components/Shared/SharedContents";
import {getFolderSampleList} from "../../utils/api";
import * as S from "../styles/Shared.style";

export default function Shared() {
  const [data, setData] = useState(null);
```
useState 만 import해서 쓰시면 더 일관된 스타일일 것 같네요!